From 86f82dc0a0b0f5baf150cb1232f92192c6146ff5 Mon Sep 17 00:00:00 2001
From: Steve deRosier <Steve.Derosier@lairdtech.com>
Date: Thu, 18 Jul 2013 18:29:05 -0700
Subject: [PATCH 6/6] Update bootstrap version to 3.4.4-laird2

Also fixed the Makefile so it won't produced versioned binary names. This
will simplify buildroot and make it not break when we update versions.
---
 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 46756d0..2aeea90 100644
--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,7 @@ BINDIR:=$(TOPDIR)/binaries
 
 DATE := $(shell date)
 VERSION := 3.4
-REVISION := 4
+REVISION := 4-laird2
 
 noconfig_targets:= menuconfig defconfig $(CONFIG) oldconfig
 
@@ -138,7 +138,7 @@ endif
 
 obj=build/$(BOARDNAME)/
 
-BOOT_NAME=$(BOARDNAME)-$(PROJECT)$(CARD_SUFFIX)boot-$(IMAGE_NAME)-$(VERSION)$(REV)
+BOOT_NAME=$(BOARDNAME)-$(PROJECT)$(CARD_SUFFIX)boot-$(IMAGE_NAME)
 AT91BOOTSTRAP:=$(BINDIR)/$(BOOT_NAME).bin
 
 ifeq ($(IMAGE),)
-- 
1.7.9.5

